flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/slotmanager/SlotManagerImpl.java

```diff
@@ -133,14 +133,17 @@
 	@Nullable
 	private final WorkerResourceSpec defaultWorkerResourceSpec;
 
+	private final int numSlotsPerWorker;
+
 	public SlotManagerImpl(
 			SlotMatchingStrategy slotMatchingStrategy,
 			ScheduledExecutor scheduledExecutor,
 			Time taskManagerRequestTimeout,
 			Time slotRequestTimeout,
 			Time taskManagerTimeout,
 			boolean waitResultConsumedBeforeRelease,
-			@Nullable WorkerResourceSpec defaultWorkerResourceSpec) {
+			@Nullable WorkerResourceSpec defaultWorkerResourceSpec,
+			int numSlotsPerWorker) {
```

Comment:
Maybe it would make sense to pass in the `SlotManagerConfiguration`. If this is possible, then one would not have to add another field to the constructor every time we add a new configuration parameter for the `SlotManager`.